SU-HSDO/suhumsci
# fix: Identify & Remove Console Errors for Font Aweseome

Closes [STN-92](https://sparkbox.atlassian.net/browse/STN-929)

# READY FOR REVIEW

## Summary
Setting `$fa-font-path: "../../humsci_basic/node_modules/@fortawesome/fontawesome-free/webfonts" !default;` so that the themes no where the fonts to be downloaded are located.

## Need Review By (Date)
asap

## Urgency
low

## Steps to Test
1. Navigate through the site (using both themes) and not only observe the absence of console errors regarding Font Awesome, but confirm that icons are still present (i.e. `/events`).

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
